_pages/policies/employee-resources-policies/employment-verification.md

```diff
@@ -16,18 +16,21 @@ tags:
 
 _How to provide verification of employment at GSA and salary_
 
-## Employment Verification
+## The Work Number is NOT currently available
```

Comment:
what is the work number that's not available, and is that what we want to lead with?

Reply:
Oh, i see below. What if we just removed that section, would that work?

Reply:
My understanding is that GSA is currently working on a new contract with The Work Number so eventually it will be available again, so I was trying to represent this is temporary AND make it easier to restore the information. That said, it is all in `git` and can be restored that way, I don't have a strong preference...what do you think?

Reply:
maybe move what you have down to where the old graf was and call it good?

Reply:
I've resolved by moving the current process down to where the old graf was, and commenting out the prior content so it'll be easy to restore if needed. Does that do the trick @tbaxter-18f ?